client/deis.py

```diff
@@ -1043,18 +1043,35 @@ def certs_add(self, args):
         """
         Binds a certificate/key pair to an application.
 
-        Usage: deis certs:add <cert> <key>
+        Usage: deis certs:add <cert> <key> [--subject-alt-name=<san>...] [options]
```

Comment:
That's possible. We could just make it a single flag called `--subject-alt-names=foo.example.com,bar.example.com` and split them up by the comma. There doesn't seem to be any arguments against that in http://pubs.opengroup.org/stage7tc1/basedefs/V1_chap12.html